### PR TITLE
fix: duplicate flag values

### PR DESCRIPTION
**Pull request by dcreey**

### CODE REVIEW

**Review by verbanicm — APPROVED** [review]

File: pkg/commands/drift/flags/driftissue.go [hunk 1]
```diff
@@ -54,7 +50,7 @@ func (d *DriftIssueFlags) Register(set *cli.FlagSet, options *Options) {
 		Target:  &d.FlagGitHubIssueLabels,
 		Example: "guardian-iam-drift",
 		Usage:   `The labels to use on any created GitHub Issues.`,
-		Default: []string{options.DefaultIssueLabel},
+		Default: []string{},
```
Comment (verbanicm) [inline pkg/commands/drift/flags/driftissue.go:53]: Should this be removed and do a nil check instead? What if someone wants zero labels to be given?

Comment (dcreey) [reply pkg/commands/drift/flags/driftissue.go:53]: That would be problematic as we manage the issues based on these labels (lookup, close, etc). I wouldn't want to add that complexity to this tool

**Review by drevell — COMMENTED** [review]
Holding off on review until Mike's comment is addressed

**Comment by dcreey** [comment]
Its been addressed

**Review by sethvargo — APPROVED** [review]

**Pull request merged**